javaslang/generator/Generator.scala

```diff
@@ -309,17 +310,20 @@ def generateMainClasses(): Unit = {
               return $mapType.ofAll(map);
           }
 
-          /$javadoc
-           * Alias for {@link $mapType#of(Object...)}
-           *
-           * @param <K>   The key type.
-           * @param <V>   The value type.
-           * @param pairs A list of key-value pairs.
-           * @return A new {@link $mapType} instance containing the given entries
-           */
-          public static <K, V> $mapType<K, V> $name(Object... pairs) {
-              return $mapType.of(pairs);
-          }
+          ${(2 to VARARGS).gen(i => {
+            xs"""
+              /$javadoc
+               * Alias for {@link $mapType#of(${(1 to i).gen(j => "K, V")(", ")})}
```

Comment:
Javadoc linter in stage `javadoc:javadoc` will be generate warnings in link to `HashMap#of(K, V, K, V)`. I think JavaDoc linter does not known about generics.
Link `HashMap#of(Object, Object, Object, Object)` will be correctly parsed by JavaDoc linter and Java IDE.